### DISCUSSION TITLE
Qiitaマイルストーンに取り上げられていない10LGTM記事がある

### DISCUSSION THREAD

**Discussion by yoshi389111** [increments/qiita-discussions#105] [要望・アイデア・不具合]
不具合なのかどうか自信がありませんが、Qiitaの記事で10LGTMに達しているにもかかわらず（認識している記事は、2021-12-29時点で9LGTM、2021-12-30時点で10LGTM）、Qiitaマイルストーンに取り上げられていない記事があるようです（2022-01-04時点）。

そもそもQiitaマイルストーンで、１件の漏れもなく紹介される前提なのか（何らかのフィルターを掛けているとか、該当記事が多い場合にはサンプリングになるとか、オプトアウトできるとか）、わかっていませんが。

もし、漏れなく紹介される前提であれば、確認をお願いしたいと思います。
逆に全量紹介の前提でないなら、その旨をどこかに明記してもらえるとありがたいです。

**Comment by kskwtnk** [comment]
@yoshi389111 
ご意見ありがとうございます！

アドベントカレンダーの時期だったのもありキューが詰まっていて、かつ年末年始で対応がもれておりました、申し訳ないです 🙇 

ちなみに基本的には全ての記事を紹介しているのですが、上記のようにキューが詰まって漏れてしまったり順番が前後することがときどきあります 🙇

**Reply by yoshi389111** [reply]
了解しました。ありがとうございます
